resources/views/users/view.blade.php

```diff
@@ -1015,23 +1015,36 @@ class="table table-striped snipe-table"
         </div><!-- /.tab-pane -->
 
         <div class="tab-pane" id="managed">
-          <div class="table-responsive">
```

Comment:
I don't know that I got this from the PR description, but switching from server-rendered lists to API-rendered lists here is a big deal. A great change, mind you! But something that should have popped on the PR description.

Reply:
I did actually mention that. "BONUS: Added manager_id to the locations API filter, which then let me switch us over to using the API for "Managed Locations" within the user detail view (no more client-side sorting, which speeds up page load quite a bit)"